### PR TITLE
Add member profile

### PR DESCRIPTION
**Pull request by fiestared**
@sot528 
- added bato and kamei's profile
- deleted kamei's twitter button (he doesn't have an account)
    - I especially want you to check the code which enable only fb button to be on profile.

### CODE REVIEW

File: src/index.pug [hunk 1]
```diff
@@ -146,6 +146,18 @@ body
...
+        include mixin/team_member
+      .row
+        .col-lg-4
+          +team_member(team.profile.erichin.name, 'Communication / Engineering')(src='/img/erichin.jpg')
+          +team_member_sns('bato.erichin', 'erichinbato')
+          .team__content
+            | #{team.profile.erichin.body}
+        .col-lg-4
+          +team_member(team.profile.kamei.name, 'Regal department')(src='/img/kamei.jpg')
+          +team_member_fb('https://www.facebook.com/tatsuhiko.kamei.9')
```
Comment (sot528) [inline src/index.pug]: [MUST]
Wrong parameter.
Please check it out.

File: src/index.pug [hunk 2]
```diff
@@ -146,6 +146,18 @@ body
           +team_member_sns('sota.ishii', 'sot528')
           .team__content
             | #{team.profile.ishii.body}
+        include mixin/team_member
```
Comment (sot528) [inline src/index.pug]: [nits]
Redundant `include`.
Please remove this line. And please move first one to correct line.

File: src/mixin/team_member_sns.pug [hunk 3]
```diff
@@ -4,3 +4,8 @@ mixin team_member_sns(fb_id, twitter_id)
       img(src='/img/facebook_logo.svg', alt='facebook')
     a(href="https://twitter.com/" + twitter_id, target="_blank")
       img(src='/img/twitter_logo.svg', alt='twitter')
+
+mixin team_member_fb(fb_id)
+  p.team__sns-link
+    a(href="https://www.facebook.com/" + fb_id, target="_blank")
+      img(src='/img/facebook_logo.svg', alt='facebook')
```
Comment (sot528) [inline src/mixin/team_member_sns.pug]: [IMO]
A little bit redundant.
You can marge it with `team_member_sns`.

ex:
```pug

mixin team_member_sns(fb_id, twitter_id)
  p.team__sns-link
    +team_member_fb(fb_id)
    a(href="https://twitter.com/" + twitter_id, target="_blank")
      img(src='/img/twitter_logo.svg', alt='twitter')

mixin team_member_fb(fb_id)
  a(href="https://www.facebook.com/" + fb_id, target="_blank")
    img(src='/img/facebook_logo.svg', alt='facebook')

```

Needed moving `p.team__sns-link` to correct place.
Let me know if it difficult! ;)

File: src/index.pug (hunk 2 above)
Comment (fiestared) [inline src/index.pug]: I see. I removed it.

File: src/index.pug (hunk 1 above)
Comment (fiestared) [inline src/index.pug]: Sorry, I missed this wrong parameter. Working late made me a lots of miss... I fixed it.

File: src/mixin/team_member_sns.pug (hunk 3 above)
Comment (fiestared) [inline src/mixin/team_member_sns.pug]: I fixed but not sure I correctly did it. Please check it.

**Comment by fiestared** [comment]
@sot528 Thanks for your review! There are always lots of things to learn from your review points. I fixed all the points you mentioned, so please check it if it's correct!

**Comment by sot528** [comment]
LGTM!!

**Pull request merged**